### ISSUE TITLE
Concerns about size

### ISSUE THREAD

**Issue by nealoke** [pladaria/react-emojione#16]
I might be wrong but is it possible to only cherry pick the emoji I want to use and leave out the rest? This due to the fact that a size of 2,5 MB is quite a lot of bandwidth for mobile users etc.

**Comment by pladaria** [comment]
Yes it is possible, this will be supported in next version. Anyway the sprite is less than 2MB and with proper cache configuration, It will be downloaded just once
